Lib/test/test_dict.py

```diff
@@ -213,7 +213,6 @@ def __next__(self):
         self.assertRaises(ValueError, {}.update, [(1, 2, 3)])
 
     # TODO: RUSTPYTHON
-    @unittest.expectedFailure
```

Comment:
When you remove this line, please also remove the `# TODO: RUSTPYTHON` line above it.